### ISSUE TITLE
mascli 2.6.1 - no permission to create 'logs' derectory when mirror images

### ISSUE THREAD

**Issue by liupinyi** [ibm-mas/ansible-devops#496]
```
4. Run Mirror Process
mkdir: cannot create directory ‘/logs/’: Permission denied
Mirroring Red Hat Release ... /logs/mirror-20221027-052818-ocp-release.log/mascli/functions/mirror_to_registry: line 366: /logs/mirror-20221027-052818-ocp-release.log: No such file or directory
[FAILURE] Red Hat Release: /logs/mirror-20221027-052818-ocp-release.log
Mirroring Red Hat Operators ... /logs/mirror-20221027-052818-ocp-operators.log/mascli/functions/mirror_to_registry: line 366: /logs/mirror-20221027-052818-ocp-operators.log: No such file or directory
[FAILURE] Red Hat Operators: /logs/mirror-20221027-052818-ocp-operators.log
Mirroring IBM Maximo Application Suite Core ... /logs/mirror-20221027-052818-core.log/mascli/functions/mirror_to_registry: line 366: /logs/mirror-20221027-052818-core.log: No such file or directory
[FAILURE] IBM Maximo Application Suite Core: /logs/mirror-20221027-052818-core.log
[SKIPPED] IBM Maximo Assist
[SKIPPED] IBM Maximo HP Utilties
[SKIPPED] IBM Maximo IoT
Mirroring IBM Maximo Manage ... /logs/mirror-20221027-052818-manage.log/mascli/functions/mirror_to_registry: line 366: /logs/mirror-20221027-052818-manage.log: No such file or directory
[FAILURE] IBM Maximo Manage: /logs/mirror-20221027-052818-manage.log
```

![image](https://user-images.githubusercontent.com/21214618/198199700-2e91a399-0e1f-454b-b128-36e3ef758bc3.png)

**Comment by liupinyi** [comment]
too many bugs.....

**Comment by durera** [comment]
This is fixed in https://github.com/ibm-mas/cli/pull/83, available since 2.7.2 :+1:

**Comment by liupinyi** [comment]
@durera Thank you for your quick fix.
